src/ResourceManager/Common/Commands.ResourceManager.Common/Paging/PageEnumerable.cs

```diff
@@ -19,44 +19,31 @@
 using System.Collections.Generic;
 using System.Threading;
 
-namespace Microsoft.Azure.Internal.Subscriptions.Models
+namespace Microsoft.Azure.Commands.ResourceManager.Common.Paging
```

Comment:
same comment - we cannot make breaking changes here.  If we need to substantially redesign it, we should define a new class.

Reply:
@markcowl this has been reverted